src/transform/src/dataflow.rs

```diff
@@ -17,6 +17,7 @@
 use std::collections::{BTreeMap, BTreeSet};
 
 use itertools::Itertools;
+
```

Comment:
```suggestion
```

Reply:
Don't we want to separate these imports?